scripts/api.js

```diff
@@ -38,6 +38,8 @@ jsdoc.explainSync( { files: JSDOC_INPUT } )
 
 function transform( v ) {
 
+	if ( v.access === 'protected' ) return;
```

Comment:
Wanted to write JSDoc comments for the "protected" methods that classes extending Controller must call in order to trigger onChange and onFinishChange. Don't want them in the docs though.